src/main/java/io/github/pmckeown/dependencytrack/finding/report/FindingsReport.java

```diff
@@ -39,6 +50,9 @@ public boolean getPolicyBreached() {
         return policyBreached;
     }
 
+    @XmlElement(name = "policyViolations")
+    public PolicyViolationsWrapper getPolicyViolationWrapper() { return new PolicyViolationsWrapper(policyViolations.size(), policyViolations); }
```

Comment:
Please format things consistently
```suggestion
    public PolicyViolationsWrapper getPolicyViolationWrapper() { 
        return new PolicyViolationsWrapper(policyViolations.size(), policyViolations); 
    }
```